.github/workflows/CI.yml

```diff
@@ -17,7 +17,7 @@ on:
 
 jobs:
   test:
-    if: (github.event_name == 'schedule' && github.repository == 'openforcefield/openforcefield') || (github.event_name != 'schedule')
+    if: (github.event_name == 'schedule' && github.repository == 'openforcefield/openff-toolkit') || (github.event_name != 'schedule')
```

Comment:
You probably already came across this, but CI will only run with an update to the `master` branch as it only runs when PRs are opened against `master` or `maintenance/.+` branches

Reply:
Thanks -- I'll try changing this line to see if I can get CI kicking again.